### PR TITLE
articles reading to the page

### PR DESCRIPTION
**Pull request by ChaseHamby**

### CODE REVIEW

File: src/App/App.js [hunk 1]
```diff
@@ -17,6 +17,8 @@ import MyNavbar from '../components/MyNavbar/myNavbar';
 import './App.scss';
 import 'bootstrap/dist/css/bootstrap.min.css';
 import authRequests from '../helpers/data/authRequests';
+import smashRequests from '../helpers/data/smashRequests';
```
Comment (morecallan) [inline src/App/App.js:20]: You shouldn't need these requests in this file. They'll happen down where state lives on those article components.

File: src/components/pages/Articles/articles.js [hunk 2]
```diff
@@ -1,11 +1,36 @@
...
+        id={article.id}
+        title={article.title}
+        synopsis={article.synopsis}
+        url={article.url}
+      />);
+    });
     return (
       <div className="Articles">
         <h2>Articles Component</h2>
+        <div>{articleBuilder}</div>
```
Comment (morecallan) [inline src/components/pages/Articles/articles.js:33]: 👍

File: src/helpers/data/messageRequests.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import axios from 'axios';
-import moment from 'moment';
+// import moment from 'moment';
```
Comment (morecallan) [inline src/helpers/data/messageRequests.js:2]: Is there a reason this is commented out? Does it not run without and 'npm install'?

**Review by morecallan — COMMENTED** [review]
@ChaseHamby - Few small things but otherwise looks good!

File: src/helpers/data/messageRequests.js (hunk 3 above)
Comment (ChaseHamby) [reply src/helpers/data/messageRequests.js:2]: Ah yes - good call. I'll comment them back in and run the npm install.

File: src/App/App.js (hunk 1 above)
Comment (ChaseHamby) [reply src/App/App.js:20]: 👍

**Pull request merged**